src/home-page/NoMatch404.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import { MainContainer } from '../layout/components/MainContainer';
+
+const NoMatchTypography = styled('div')`
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction:column;
+  margin: auto;
+  min-height: 70vh;
+`;
+
+const HeaderTypography = styled(Typography)`
+  font-size:10em;
+`;
+const SubtitleTypography = styled(Typography)`
+  font-size:2em;
+  margin: ${props => props.theme.spacing(2)}px;  
+`;
+
+const StyledButton = styled(Button)`
+color: white;
+`;
+
+export const NoMatch404: React.FC<{}> = () => {
+  const history = useHistory();
+
+  return (
+    <MainContainer>
+      <NoMatchTypography>
+        <HeaderTypography variant="h1">404</HeaderTypography>
```

Comment:
Sorry, I didn't notice it before> I think it shuldn't be h1. H1 should be reserved for the main information on our page, like the main heading.

Reply:
done;)